install/rhel-installation-guide.md

```diff
@@ -1,19 +1,11 @@
 # Install Kata Containers on RHEL
 
-> **Warning:**
->
-> - The RHEL packages are provided as a convenience to users until native
->   packages are available in RHEL. However, they are **NOT** currently tested
->   (although CentOS is) so caution should be exercised.
->
->   See https://github.com/kata-containers/ci/issues/3 for further details.
-
```

Comment:
I'd like to see this warning removed, but aren't we still blocked on the following?

- https://github.com/kata-containers/ci/issues/3
- https://github.com/kata-containers/ci/issues/129

Reply:
@jodh-intel , I already open a PR in tests for installation scripts, I am waiting there for a review

Reply:
OK. This needs to block on it then, so I'll ad a dnm label...